### ISSUE TITLE
Browser extension to choose font and font size of code blocks on websites

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/module-requests#97]
Would be useful with a Chrome extension where you can choose your favorite font and size and have it apply to code blocks on any website. Would make reading code on the web much more enjoyable as you can read it just like you would in your editor.

**Comment by FezVrasta** [comment]
Isn't [Stylish](https://userstyles.org/) enough?

**Comment by sindresorhus** [comment]
You could achieve the same, sure, but it's not very user-friendly.

**Comment by bfred-it** [comment]
I agree with @FezVrasta; too much overhead for what essentially should be:

```css
code, pre {
	font-family: 'Courier New'; /* ❤️ */
}
```

**Comment by benjamin-rood** [comment]
^ Just an fyi on the above styling:
The font naming of the default Fira Code font family specification in CSS styling will be `Fira Code` (at least on macOS, but IIRC it's true regardless), whereas any variant e.g. _Light_ will be `FiraCode-Light`. This is the same for all browser-like apps such as Visual Studio Code.

**Comment by ssight** [comment]
I've made something similar to this, [https://ssight.github.io/CodeText/](CodeText), although it doesn't work very well (you have to sideload the extension it provides) - but it _does_ do the job.

Maybe it's what you're looking for?

**Comment by ssight** [comment]
@sindresorhus @fregante I've created an extension for Chrome called CodeText, and I think it does the job fairly easily. You'll just have to download it from the releases section.

If anybody wants to upload it to the Chrome Web Store, feel free, I'll really appreciate it.

**Comment by vaibhavhrt** [comment]
@ssight sorry can't seem to find it anymore.

**Comment by ssight** [comment]
@vaibhavhrt Hm, it seems that it wasn't on GitHub anymore. Nevermind, I've `git push`ed it again and back it's on now.

**Comment by vaibhavhrt** [comment]
cool I will install it & check it out.
Also I have chrome extension dev so I can publish it on chrome store
